source/viz.md

```diff
@@ -1281,27 +1346,28 @@ Histogram of Michelson's speed of light data colored by experiment.
 ```{index} integer
 ```
 
-Alright great, {numref}`final_plot_colored` looks...wait a second! We are not able to distinguish 
-between different Experiments in the histogram! What is going on here? Well, if you 
+Alright great, {numref}`final_plot_colored` looks...wait a second! We are not able to distinguish
+between different Experiments in the histogram! What is going on here? Well, if you
 recall from Chapter {ref}`wrangling`, the *data type* you use for each variable
 can influence how Python and `altair` treats it. Here, we indeed have an issue
 with the data types in the `morley` data frame. In particular, the `Expt` column
 is currently an *integer*. But we want to treat it as a
-*category*, i.e., there should be one category per type of experiment.  
+*category*, i.e., there should be one category per type of experiment.
 
 ```{index} nominal, altair; :N
 ```
 
-To fix this issue we can convert the `Expt` variable into a `nominal`(categorical) type 
+To fix this issue we can convert the `Expt` variable into a `nominal`(categorical) type
 variable by adding a suffix `:N`(where `N` stands for nominal type variable) with the `Expt` variable.
 By doing this, we are ensuring that `altair` will treat this variable as a categorical variable,
 and the color will be mapped discretely. Here, we also mention `stack=False`, so that the bars are not stacked on top of each other.
 
 ```{code-cell} ipython3
 morley_hist_categorical = alt.Chart(morley_df).mark_bar(opacity=0.5).encode(
-    x = alt.X("Speed", bin=alt.Bin(maxbins=50)),  
-    y=alt.Y('count()', stack=False),
-    color = "Expt:N")
+    x=alt.X("Speed", bin=alt.Bin(maxbins=50)),
+    y=alt.Y("count()", stack=False),
+    color="Expt:N"
```

Comment:
This `:N` thing is neat -- is it the standard way of accomplishing this in altair? seems a bit too "magical" to me in comparison with other things, but I'm no expert so I'm not sure.

Reply:
this is perhaps a question for @joelostblom?? Altair is fairly new to me too. Any suggestions appreciated!

Reply:
Looks like it is standard:

https://altair-viz.github.io/user_guide/encoding.html#encoding-data-types

in that case, @lheagy it would probably be a good idea to provide a table summarizing the different types and codes for them (which appears on that webpage I linked)